src/shipit_signoff/README.md

```diff
@@ -4,7 +4,8 @@
     pip install -r requirements.txt -r requirements-dev.txt -c requirements_frozen.txt 
     pip install -e . -c requirements_frozen.txt 
     Fill in `client_secrets.json.tmpl` as `client_secrets.json` for auth0 connections
-    python3 shipit_signoff/__init__.py
+    Set the environment variables APP_URL DATABASE_URL
```

Comment:
While the documentation is less likely to get stale if we just tell people to look at `settings.py`, it also increases the barrier to entry.

I agree about having a common place for a description of how to start development.